### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  createIfNotExists(path);
+  fs.readFile(path, (err, commands) => {
+    if (err) throw err;
+    const parseJson = JSON.parse(commands);
+    parseJson.push(newComm);
+    fs.writeFile(
+      path,
+      JSON.stringify(parseJson),
+      err => {
+        if (err) throw err;
```
Comment (tshemsedinov) [inline commands.js]: Need a consistent error handling, in line 14 you use `console.log`, here you throws...

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
+module.exports = {
```
Comment (tshemsedinov) [inline .eslintrc.js:2]: this eslint config is not checked by eslint )))

**Review by tshemsedinov — COMMENTED** [review]
Very short

**Comment by ShpylovyiRoman** [comment]
I close this PR due to the fact, that everything is resolved in another PRs (except for 'very short' of course :)

**Pull request closed without merging**